server/controllers/articles.js

```diff
@@ -35,11 +24,7 @@ class allAboutArticle {
       flag: 0,
     };
     articles.push(data);
-    return res.status(201).json({
-      status: '201',
-      message: 'article successfuly created',
-      data,
-    });
+    returnFc(req, res, '201', 'article successfuly created', data);
```

Comment:
@tuyisengepaul  i do not understand what you mean by `returnFc()` please make your functions more descriptive

Reply:
This is the helper function to handle all responses.

Reply:
Resolved